docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
```

Comment:
Also this should end with a colon, and perhaps explain a little about why this is necessary.